### PR TITLE
Renamed lead space wrapper

### PR DESCRIPTION
**Pull request by photodow**
### Related Ticket(s)

#1028

### Changelog

**Changed**

- renamed lead space wrapper

### CODE REVIEW

File: wiki/Layout-component/Lead-space-search.md [hunk 1]
```diff
@@ -15,7 +15,6 @@ The [[Lead space search]] is in the same family of our other Lead space componen
 - [[Search typeahead]]
 - [[Tag group]]
 - [[Carbon breadcrumb]]
-- [[Lead space wrapper]]
```
Comment (shixiedesign) [inline wiki/Layout-component/Lead-space-search.md:18]: Wouldn't Lead space search need `Background media` too?

Comment (photodow) [reply wiki/Layout-component/Lead-space-search.md:18]: I ended up removing it here because it doesn't depend on it yet.

Comment (shixiedesign) [reply wiki/Layout-component/Lead-space-search.md:18]: Ah yes make sense! TY

**Review by shixiedesign — APPROVED** [review]
Looking good. Thanks!

**Pull request merged**